packages/create-vite/template-ember-ts/src/app.ts

```diff
@@ -0,0 +1,10 @@
+import Application from '@ember/application'
+import Resolver from 'ember-resolver'
+import config from '#config'
+
+import { registry } from './registry.js'
```

Comment:
Is this `.js` wanted or a copy paste from JS template?

Reply:
it should be .ts, thank you.
it 'works' (cause vite's cool like that)
but it should be ts to match source
```suggestion
import { registry } from './registry.ts'
```